### ISSUE TITLE
bind mounts sometimes mounting as XFS

### ISSUE THREAD

**Issue by jakerundall** [ncsa/puppet-spectrumscale#9]
We found some GPFS bind mounts mounted as XFS:
...
/dev/mapper/system-root on /scratch type xfs (rw,relatime,attr2,inode64,noquota)
/dev/mapper/system-root on /software type xfs (rw,relatime,attr2,inode64,noquota)
/dev/mapper/system-root on /datasets type xfs (rw,relatime,attr2,inode64,noquota)
/dev/mapper/system-root on /home type xfs (rw,relatime,attr2,inode64,noquota)
/dev/mapper/system-root on /project type xfs (rw,relatime,attr2,inode64,noquota)
/dev/mapper/system-root on /dedicated type xfs (rw,relatime,attr2,inode64,noquota)
tmpfs on /run/user/0 type tmpfs (rw,nosuid,nodev,relatime,size=13174588k,mode=700)
fs0 on /gpfs/fs0 type gpfs (rw,relatime)

This occurred on a machine running CentOS 7.4 after its GPFS client RPMs were updated and Puppet tried to build new kernel modules, etc.

We discussed and decided it would be worth making the following change in manifests/bindmount.pp:
`    # Resource defaults
    Mount {
        ensure  => mounted,
        fstype   => 'none',
    }
`
-->
`    # Resource defaults
    Mount {
        ensure  => mounted,
        fstype   => 'gpfs',
    }
`
In other words, update the fstype from 'none' to 'gpfs'. I tested this change and was able to remount as GPFS after making that change. I'm not entirely sure how to test whether or not it would prevent the actual issue of mounting as XFS because I'm not entirely sure why it happened. But it seems like it won't hurt and might help.

Jake

**Comment by andylytical** [comment]
fixed in v1.0
